standard/documentation-comments.md

```diff
@@ -316,13 +325,16 @@ where
 **Example:**
 
 ```csharp
-/// <summary>This method changes the point's location to
-/// the given coordinates.</summary>
+/// <summary>
+/// This method changes the point's location to
+/// the given coordinates.
+/// </summary>
 /// <param name="xor">the new x-coordinate.</param>
 /// <param name="yor">the new y-coordinate.</param>
-public void Move(int xor, int yor) {
-   X = xor;
-   Y = yor;
+public void Move(int xor, int yor)
```

Comment:
Again, possible parameter name changes - but again, potentially outside the scope of this PR.

Reply:
I went ahead and changed them.